interna/crowdfund/tests/test_views.py

```diff
@@ -52,3 +52,11 @@ def test_status_codes(client, method, url, pk, user, status_code):
     reversed_url = reverse(url) if pk is None else reverse(url, kwargs={'pk': pk})
     response = getattr(client, method.lower())(reversed_url)
     assert response.status_code == status_code
+
+
+@pytest.mark.django_db
+def test_pledge_name(client):
+    prj = mommy.make(models.Project)
+    response = client.get('/crowdfund/projects/{}/'.format(prj.pk))
+    assert response.status_code == 200
+    #import ipdb; ipdb.set_trace()
```

Comment:
I assume this is WIP :)

You can find the reference for the test client here: https://docs.djangoproject.com/en/2.0/topics/testing/tools/#the-test-client Apparently you can set cookies on [client.cookies](https://docs.djangoproject.com/en/2.0/topics/testing/tools/#django.test.Client.cookies).

Reply:
Yes, thanks :)